### PR TITLE
STAC-19602: don't rewrite openmetrics histogram

### PR DESCRIPTION
**Pull request by fvlankvelt**
### Step 3: Did you add / update tests for your changes in the right area?
- [ ] Unit/Component test
- [ ] Integration test	

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: stackstate_checks_dev/tests/docker/test_default.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
 
   vault:
-    image: artifactory.tooling.stackstate.io/docker-virtual/vault:latest
+    image: hashicorp/vault:latest
```
Comment (LouisParkin) [inline stackstate_checks_dev/tests/docker/test_default.yaml:6]: artifactory.tooling.stackstate.io/docker-virtual/hashicorp/vault:latest

Comment (fvlankvelt) [reply stackstate_checks_dev/tests/docker/test_default.yaml]: good catch!

File: openmetrics/tests/test_openmetrics.py [hunk 2]
```diff
@@ -67,33 +70,49 @@ def test_openmetrics_check(aggregator):
...
-    instance['send_monotonic_counter'] = False
-    c = OpenMetricsCheck('openmetrics', None, {}, [instance])
-    c.check(instance)
     aggregator.assert_metric(
-        CHECK_NAME + '.renamed.metric1',
-        tags=['node:host1', 'flavor:test', 'matched_label:foobar'],
-        metric_type=aggregator.GAUGE
+        CHECK_NAME + '.hist1.bucket',
+        tags=['node:host2', 'le:0.0'],
+        metric_type=aggregator.GAUGE,
```
[the remark this answers is not in the document]
Comment (fvlankvelt) [reply openmetrics/tests/test_openmetrics.py:78]: sharp - hadn't thought about this, but the values are indeed counts.  AFAICS, gauges get the "leave it alone"  treatment in the rest of the code; am not so sure about counters.  Let me try that out.

[the remark this answers is not in the document]
Comment (craffit) [reply openmetrics/tests/test_openmetrics.py:78]: another question on this: are the value we report really gauges or counters? (If they are gauges, this means we have to un-gauge them to be cumulative counters)

[the remark this answers is not in the document]
Comment (fvlankvelt) [reply openmetrics/tests/test_openmetrics.py:78]: I think its best to stay as close as possible to prometheus defaults / just pass through whatever metrics are reported.  Based on the histogram values, promql queries (like `rate` and `histogram_quantile`) can turn the metrics into the values we want to report / graph.

[the remark this answers is not in the document]
Comment (fvlankvelt) [reply openmetrics/tests/test_openmetrics.py:78]: I've not been able to find a counter that's actually collected.  So there's something fishy going on with that too.

**Pull request merged**